### ISSUE TITLE
How to filter file in subfolder 

### ISSUE THREAD

**Issue by fvn-manhtd** [trippo/ResponsiveFilemanager#487]
Hi everyone,

Please show me how to filter file in subfolder?
Currently, Responsive File manager only can filter in parent folder.

<img width="859" alt="screen shot 2018-08-09 at 19 36 33" src="https://user-images.githubusercontent.com/20807013/43893978-b3812622-9c0b-11e8-9f6b-ca8210be53cf.png">

**Comment by trippo** [comment]
Yes because rfm don’t have a database to store files only show current folder

**Comment by Ruben-gzraga** [comment]
@trippo could you tell me how could i do this with a database in rfm (sorry about my english).  Thanks in advance
